modules/sdk/project-templates/project-templates/src/test/java/com/liferay/project/templates/ProjectTemplatesTest.java

```diff
@@ -4024,7 +4024,7 @@ else if (key.equals("liferayVersion")) {
 		projectTemplatesArgs.setService(service);
 		projectTemplatesArgs.setTemplate(template);
 
-		Archetyper archetyper = new Archetyper();
+		ProjectGenerator archetyper = new ProjectGenerator();
```

Comment:
The variable name should match the class name of the object when possible for code readability.